pages/add/add.scss

```diff
@@ -1,187 +1,143 @@
 .add-page {
```

Comment:
A lot of the following rules has been 

- moved to `scss/forms.scss`
- or replaced by assigning boostrap classes

Reply:
I think as long as the /add page still looks legit we are good.